spec/system/admin_customize_themes_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+describe 'Admin Customize Themes', type: :system, js: true do
+  fab!(:color_scheme) { Fabricate(:color_scheme) }
+  fab!(:theme) { Fabricate(:theme) }
+  fab!(:admin) { Fabricate(:admin) }
+
+  before do
+    sign_in(admin)
+  end
+
+  describe 'when visiting the page to customize the theme' do
+    it 'should allow admin to update the color scheme of the theme' do
+      visit("/admin/customize/themes/#{theme.id}")
+
+      within(".theme-settings__color-scheme") do
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
```

Comment:
Same question for this, is it supposed to be in the `within()` block?

Reply:
Im not exactly sure how `within()` works but I expect it to scope for things like `find` or `fillIn` but most likely not for `expect(page)`.

Reply:
O you're right, `within` does nothing for page expectations or even page lookups